### ISSUE TITLE
Can't scroll past Flickity on touch device

### ISSUE THREAD

**Issue by latwelve** [metafizzy/flickity#97]
Flickity works fine - but I can't swipe passed it. It takes up the full viewport (which I want it to) but up and down swiping looks to be bound to the left/right slide navigating of flickity so there is no way to get to the content on the page below?

**Comment by desandro** [comment]
I'm sorry to see you're having trouble with Flickity. Could you provide a **reduced test case**? See [Submitting Issues in the contributing guidelines](https://github.com/metafizzy/flickity/blob/master/contributing.md#submitting-issues).

**Comment by latwelve** [comment]
Thanks Desandro - http://codepen.io/anon/pen/Bybzxb

So the issue is on an iPhone once you've scrolled down to where flickity sits I can navigate flickity absolutely fine - I just can't scroll past. There are no dom elements surrounding it that I can drag over.

Ideally I'm hoping for any vertical swipe allowing the user to continue down the dom (or up)

Thanks

**Comment by evoactivity** [comment]
This is down to the touchVerticalScroll and will be addressed by #67 I can't wait until that is fixed as it's causing some serious issues on a current project I'm working on.

**Comment by latwelve** [comment]
Thanks Evo - what solution are you going for - a different lib?

**Comment by desandro** [comment]
Thanks for the demo. Looks like CodePen has a bug where demo/live views cannot have their scroll set. But you can see a proper demo in the debug view. See debug CodePen http://codepen.io/desandro/debug/XJGNyr

On this demo, I'm able to scroll. On your demo, you have set `touchVerticalScroll: false`, which will specifically disable touch vertical scrolling.

**Comment by evoactivity** [comment]
@latwelve I've seen other people mention peppermint.js http://wd.dizaina.net/en/scripts/peppermint/
I'm finding the watchCSS option incredibly useful for what i'm doing though, so right now I'm hoping the feature will be addressed before I need to push into production.

**Comment by latwelve** [comment]
Thanks for the replies guys

@desandro  - that was my mistake for the demo - I removed that line and it worked fine until I then added this back into my document:

html,body
{
    width: 100%;
    height: 100%;
    margin: 0px;
    padding: 0px;
    overflow-x: hidden;
}

Which then led me to trying @evoactivity fix which works a charm - many thanks

**Comment by lisamayo** [comment]
Having the same exact same issue w.vertical swipe not working -- thanks @evoactivity for the fix!

**Comment by latwelve** [comment]
Flickity also seems very laggy on my phone now on an extremely light page.. think I'll have to try another slider for my needs for now

**Comment by desandro** [comment]
Flickity `v1.0.1` has been released with fixes for touch scrolling. [Try it out](http://flickity.metafizzy.co) and holler if this problem persists.

**Comment by kukac7** [comment]
Windows Phone (8.1) on the problem still exists. :sob:

**Comment by thulank** [comment]
me too (windows phone 8.1 + latest flickity) can't swipe verticaly

**Comment by mstrelan** [comment]
For me this only seems to occur if you're touching the previous/next arrows while scrolling.

**Comment by piotrkulpinski** [comment]
Same issue here. Can't scroll through flickity slider

**Comment by anthony0579** [comment]
@desandro Flickity is by far the best carousel i've used and implemented on a few sites. Really top draw stuff and please keep producing quality things like this. User experience is a dying art and I love things that work how they should! However, I do have this niggling problem that when trying to scroll vertically it seems just a bit too sensitive and if not a precise vertical scroll it begins to scroll horizontally.

I'm using the latest Flickity v1.2.1.

Many thanks!

**Comment by desandro** [comment]
@anthony0579 Thanks for the kind words. I'm looking to resolve this in #343 in the next version of Flickity.

**Comment by shakyShane** [comment]
@desandro This will be extremely welcomed.

We are lucky enough to have a device lab in our office and can confirm that 'flicking' past a carousel is quite the challenge.

The PR in this thread is perfect, setting the `dragThreshold` to just `10` proved enough for us 👍

**Comment by sashabeep** [comment]
touchVerticalScroll:false still not preventing scrolling on android 5.1 chrome latest, firefox

**Comment by xpd** [comment]
Got a bug here where it is impossible to scroll past the first image in a slider. dragThreshold seems to work from all the others.

**Comment by sashabeep** [comment]
BUMP

**Comment by valootech** [comment]
Hi, I had a problem with Flickity on Windows Phone device.
When you have full page height carousel, it's imposible to scroll down the page.

Fixed it via CSS: 

```
.flickity-enabled.is-draggable .flickity-viewport{
    touch-action: pan-y !important;
}
```

This hack is for rewrite inline rule style="touch-action: none;" on this block.

Works for me. May be there is a better solution?

**Comment by desandro** [comment]
@va-loo This is likely related to a recent regression. See issue #560

**Comment by davidhellmann** [comment]
@va-loo thanks. that works fine!

**Comment by winstef** [comment]
@va-loo thanks. this is the only fix that worked for me.

**Comment by akashnigam010** [comment]
@va-loo this is gold.
Struggled for a quite some time before trying your hack. Work perfectly!

@desandro , I tried version 1.2.1. This issue was taken care in that, but the performance wasn't good.
Latest version's(v2.0.8) performance is far better but this issue is again present in it. Had to go with va-loo's fix to make it work. 
Is there a configurable property that takes care of this in the latest version?

**Comment by zerodburn** [comment]
If I run Google Chrome, enable Dev Tools, enable mobile emulation and use the default iPhone 6 it is literally impossible to touch and drag down the page. Surely this is a bug?

**Comment by desandro** [comment]
I'm closing this issue as resolved. This bug has been fixed in Flickity v2.0.9. Please upgrade.

If you continue to run into this issue, please **open a new issue**. This thread has gotten a bit long over the years 😄

**Comment by sterswift** [comment]
@va-loo Saved me, thanks
